packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/contact.controller.js

```diff
@@ -381,6 +381,12 @@ export default class TelecomTelephonyServiceContactCtrl {
     if (this.directoryForm.legalForm !== 'individual') {
       this.directoryForm.PJSocialNomination = this.directoryForm.socialNomination;
     }
+
+    this.directoryForm.wayNumberExtra = this.directoryForm.wayNumberExtra.replace(
+      /&nbsp;/g,
```

Comment:
Looks like tricky workaround given the purpose. Could we fix the select behaviour instead ?

Reply:
I tried, but obviously the empty field selections were not optimal when I tried to modify ui-kit (min-height: 28px;)